pycode/memilio-simulation/memilio/simulation/bindings/simulation.cpp

```diff
@@ -141,7 +124,7 @@ PYBIND11_MODULE(_simulation, m)
             return std::vector<std::pair<mio::Date, mio::Date>>(h.begin(), h.end());
         },
         py::arg("state_id"), py::arg("start_date") = mio::Date(std::numeric_limits<int>::min(), 1, 1),
-        py::arg("end_date") = mio::Date(std::numeric_limits<int>::max(), 1, 1));
+        py::arg("end_date") = mio::Date(std::numeric_limits<int>::max(), 1, 1)); // TODO: Python doesnt know theses default values
```

Comment:
can you elaborate what exactly doesn't work? does this need a new issue?

Reply:
The old stub generator could not interpret the value of the instance of mio::Date(...). 
But with moving to mypy as the stub generator, we get no default values at all in the stubs of the c++ binded code. I didn't find a direct solution, that didn't include adjusting the stubs by hand after being generated.

Reply:
So is this related to the TODO in generate_stubs.py? 

as a general rule, every TODO comment in the code should have a corresponding issue in github to increase the chance of the TODO actually being done. Maybe one issue for all the TODOs is enough here if they are related?